hw/top_earlgrey/fpv/top_earlgrey_fpv_cfgs.hjson

```diff
@@ -0,0 +1,148 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+
+  // This is the master cfg hjson for FPV. It imports ALL individual FPV
+  // cfgs of the IPs and the full chip used in top_earlgrey. This enables to run
+  // them all as a regression in one shot.
+  name: top_earlgrey_batch_fpv
+
+  import_cfgs: [// common server configuration for results upload
+                "{proj_root}/hw/data/common_project_cfg.hjson"]
+
+  use_cfgs: [// TODO: implement some switch to only select "_fpv" testbenches
+             // TODO: if we default "_fpv" cov to be on, and the rest of the tbs cov off, need a
+             // command-line switch to disable cov.
```

Comment:
@sriyerg @msfschaffner thank you for your suggestions, and I deleted the "dedicated cfgs" and combined them into one batch_cfg. These are the only items left TODO. Currently If I only wanted to run dedicated cfgs, I just use --selected-cfgs switch.

Reply:
if we need a batch list that is only for dedicated FPV testbenches, I think it's ok to have a separate list for them and include them in the main master config. I think my comment was more related to the actual location of the config in the repo. how about we keep one master batch config `top_earlgrey_fpv_cfgs.hjson` and a dedicated config `top_earlgrey_dedicated_fpv_cfgs.hjson` under `hw/top_earlgrey/fpv` (and the latter can be included in the former)?

Reply:
Thanks for clarifying that. I had error importing `top_earlgrey_dedicated_fpv_cfgs.hjson` tp `top_earlgrey_fpv_cfgs.hjson`. That gives me a compile error saying "name" conflict. I do not think they support importing a master cfg like that.
I think the original solution is pretty messy, and the only reason we separate them is because of coverage. If we can just do a default cov, or have something like a `-i` for dedicated IPs, maybe that is a cleaner way?

Reply:
Ah ok I thought imports of master configs where working in #1854. However, if this does not work, I'd also be in favor of keeping just one list to avoid duplication. Would the default cov turn on coverage for dedicated FPV testbenches only?

Reply:
Yeah, importing cfgs only works as https://github.com/lowRISC/opentitan/pull/1854, but as you pointed out, we have to create a duplicated "formal/data/dedicated_fpv_cfgs", then import that to both "top_earlgrey_dedicated_cfgs" and "top_earlgrey_batch_cfgs". Yeah not a very ideal way to have so many duplicated configs.

And for the coverage: yes, by default we want to only turn on "dedicated FPV testbenches". However, if we turn them on in the cfg.hjson, we are not able to turn them off in the command line. This is because `-cov` is declared as `set_true` in dvsim.sv.

Reply:
sounds like we need something like a `-nocov` switch ;)... let's ask @sriyerg whether that would be ok or whether he has another idea to address this issue. 

however, I think we need not gate this PR on that decision.

Reply:
Lets think about this more in a later PR. File an issue to track this.